bciers/apps/registration/app/components/datagrid/models/operationColumns.ts

```diff
@@ -0,0 +1,40 @@
+import { GridColDef, GridRenderCellParams } from "@mui/x-data-grid";
+import StatusStyleColumnCell from "@bciers/components/datagrid/cells/StatusStyleColumnCell";
+
+export const OPERATOR_COLUMN_INDEX = 1;
+
+const operationColumns = (
+  isInternalUser: boolean,
+  ActionCell: (params: GridRenderCellParams) => JSX.Element,
+) => {
+  const columns = [
+    {
+      field: "name",
+      headerName: "Operation Name",
+      width: isInternalUser ? 720 : 320,
+    },
+    { field: "bcghg_id", headerName: "BC GHG ID", width: 200 },
+    { field: "type", headerName: "Operation Type", width: 200 },
+    {
+      field: "action",
+      headerName: "Action",
+      renderCell: ActionCell,
+      sortable: false,
+      width: 120,
+      // Set flex to 1 to make the column take up all the remaining width if user zooms out
```

Comment:
It would look like this (I'd have to fix the border): 
![image](https://github.com/bcgov/cas-registration/assets/77306764/45b7627e-63d8-447e-a605-3e1b344a89b7)

Reply:
Can't you add flex 1 to Operation Name column since it is the big one in the design?